extensions/amp-story-audio-sticker/0.1/amp-story-audio-sticker.js

```diff
@@ -157,14 +159,79 @@ export class AmpStoryAudioSticker extends AMP.BaseElement {
       : FALLBACK_DEFAULT_STICKER;
   }
 
+  /**
+   * @private
+   */
+  maybeInitializeStickerStyle_() {
+    this.toggleStickerState_(
+      'outline',
+      this.element.getAttribute('style') === 'outline'
+    );
+    this.toggleStickerState_(
+      'dropshadow',
+      this.element.getAttribute('style') === 'dropshadow'
+    );
+
+    const outlineColor = this.element.getAttribute('outline-color');
+    if (this.isValidRGBColor_(outlineColor)) {
+      setImportantStyles(this.element, {
+        '--story-audio-sticker-outline-color': outlineColor,
+      });
+    }
+    const dropshadowColor = this.element.getAttribute('dropshadow-color');
+    if (this.isValidRGBColor_(dropshadowColor)) {
+      setImportantStyles(this.element, {
+        '--story-audio-sticker-dropshadow-color': dropshadowColor,
+      });
+    }
+  }
+
+  /**
+   * @private
+   * @param {string} RGBColor
+   * @return {boolean}
+   */
+  isValidRGBColor_(RGBColor) {
+    return RGBColor && RGBColor.match(/rgba?\((\d{1,3}), (\d{1,3}), (\d{1,3})/);
+  }
+
   /** @private */
   initializeListeners_() {
     this.element.addEventListener(
       'click',
       () => this.storeService_.dispatch(Action.TOGGLE_MUTED, false),
       true
     );
-    // TODO: add listeners for click animations.
+    this.element.addEventListener(
+      'touchstart',
+      () => this.toggleStickerState_('clicking', true),
+      {passive: true}
+    );
+    this.element.addEventListener(
+      'mousedown',
```

Comment:
`pointerdown` and `pointerup` are platform agnostic ways of dealing with these, so you can use these and replace 4 listeners with just two. Also, I wonder if the `:focus` or `:focus-visible` CSS selectors can apply these behaviors instead of having to rely on JS listeners.

Reply:
The two events work! 

Using focus is an interesting direction! However, I tried tabindex and button but they both have the issue of seconds of delay before receiving focus event, which is caused by browser differentiating it is a click event or a click-and-hold event. From the UX requirement, we need more immediate shirking animation as long as the sticker is clicked.